.travis.yml

```diff
@@ -51,7 +51,7 @@ before_install:
 
 before_script:
   - npm install || exit 1
-  - composer install
+  - docker run --rm -v "$PWD:/app" composer install
   - |
     if [[ "$PHP" == "1" ]] || [[ "$JS" == "1" ]] || [[ "$SNIFF" == "1" ]]; then
       npm install -g gulp-cli
```

Comment:
These things should run in the `install` phase of the build rather than `before_script` I think